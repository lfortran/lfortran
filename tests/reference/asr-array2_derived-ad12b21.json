{
    "basename": "asr-array2_derived-ad12b21",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/array2_derived.f90",
    "infile_hash": "e369ff08e97bdc709b67c7ba894c208aebac6f07f89726c76925733c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-array2_derived-ad12b21.stdout",
    "stdout_hash": "4211cb077471ce7d30e02c4d67d392854381417a8837a6dc2e0c8f95",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}