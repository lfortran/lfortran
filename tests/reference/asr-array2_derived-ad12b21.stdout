(TranslationUnit (SymbolTable 1 {array2_derived: (Program (SymbolTable 2 {a: (Variable 2 a Local () () Default (Real 4 [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 5 (Integer 4 [])))]) Source Public Required .false.), b: (Variable 2 b Local () () Default (Real 4 [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 5 (Integer 4 [])))]) Source Public Required .false.), c: (Variable 2 c Local () () Default (Integer 4 [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 3 (Integer 4 [])))]) Source Public Required .false.), d: (Variable 2 d Local () () Default (Logical 4 [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 2 (Integer 4 [])))]) Source Public Required .false.), der_type: (DerivedType (SymbolTable 3 {i: (Variable 3 i Local () () Default (Integer 4 []) Source Public Required .false.), r: (Variable 3 r Local () () Default (Real 4 []) Source Public Required .false.)}) der_type [r i] Source Public ()), e: (Variable 2 e Local () () Default (Real 4 [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 2 (Integer 4 []))) ((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 3 (Integer 4 [])))]) Source Public Required .false.), f: (Variable 2 f Local () () Default (Integer 4 [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 3 (Integer 4 []))) ((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 4 (Integer 4 [])))]) Source Public Required .false.), g: (Variable 2 g Local () () Default (Logical 4 [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 5 (Integer 4 []))) ((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 2 (Integer 4 [])))]) Source Public Required .false.), h: (Variable 2 h Local () () Default (Real 4 [((IntegerConstant 2 (Integer 4 [])) (IntegerConstant 2 (Integer 4 []))) ((IntegerConstant 3 (Integer 4 [])) (IntegerConstant 2 (Integer 4 []))) ((IntegerConstant 4 (Integer 4 [])) (IntegerConstant 2 (Integer 4 [])))]) Source Public Required .false.), i: (Variable 2 i Local () () Default (Integer 4 [((IntegerConstant 3 (Integer 4 [])) (IntegerConstant 2 (Integer 4 []))) ((IntegerConstant 4 (Integer 4 [])) (IntegerConstant 2 (Integer 4 []))) ((IntegerConstant 3 (Integer 4 [])) (IntegerConstant 2 (Integer 4 [])))]) Source Public Required .false.), j: (Variable 2 j Local () () Default (Logical 4 [((IntegerConstant 5 (Integer 4 [])) (IntegerConstant 2 (Integer 4 []))) ((IntegerConstant 2 (Integer 4 [])) (IntegerConstant 2 (Integer 4 []))) ((IntegerConstant 2 (Integer 4 [])) (IntegerConstant 3 (Integer 4 [])))]) Source Public Required .false.), p: (Variable 2 p Local () () Default (Derived 2 der_type [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 7 (Integer 4 [])))]) Source Public Required .false.), q: (Variable 2 q Local () () Default (Derived 2 der_type [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 7 (Integer 4 []))) ((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 9 (Integer 4 [])))]) Source Public Required .false.), r: (Variable 2 r Local () () Default (Derived 2 der_type [((IntegerConstant 7 (Integer 4 [])) (IntegerConstant 4 (Integer 4 []))) ((IntegerConstant 9 (Integer 4 [])) (IntegerConstant 7 (Integer 4 []))) ((IntegerConstant 16 (Integer 4 [])) (IntegerConstant 3 (Integer 4 [])))]) Source Public Required .false.), x: (Variable 2 x Local () () Default (Complex 4 [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 6 (Integer 4 [])))]) Source Public Required .false.), y: (Variable 2 y Local () () Default (Complex 4 [((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 6 (Integer 4 []))) ((IntegerConstant 1 (Integer 4 [])) (IntegerConstant 8 (Integer 4 [])))]) Source Public Required .false.), z: (Variable 2 z Local () () Default (Complex 4 [((IntegerConstant 6 (Integer 4 [])) (IntegerConstant 2 (Integer 4 []))) ((IntegerConstant 8 (Integer 4 [])) (IntegerConstant 3 (Integer 4 []))) ((IntegerConstant 11 (Integer 4 [])) (IntegerConstant 3 (Integer 4 [])))]) Source Public Required .false.)}) array2_derived [] [])}) [])
